Caching/Sources/Caching/UserModel.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  File.swift
+//
+//
+//  Created by ziad on 11/08/2022.
+//
+
+import UIKit
+
+struct User: Codable {
+    var name: String?
+    var id: Int?
```

Comment:
Immutable properties are preferred. 
```suggestion
    let name: String?
    let id: Int?
```